reconcile/gitlab_fork_compliance.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from gitlab import MAINTAINER_ACCESS
+
+from reconcile import queries
+from utils.gitlab_api import GitLabApi
+from utils.gitlab_api import MRState
+
+
+_LOG = logging.getLogger(__name__)
+
+QONTRACT_INTEGRATION = 'gitlab-fork-compliance'
+
+MAINTAINERS_GROUP = 'app-sre'
+BOT_USER = 'devtools-bot'
+BLOCKED_LABEL = 'blocked/devtools-bot-access'
```

Comment:
let's take this opportunity to change the label to `blocked/bot-access` (i already added the label in the project)